src/camera.cpp

```diff
@@ -131,7 +131,9 @@ void Camera::OnMouseMotion(const SDL_MouseMotionEvent & event)
   }
 }
 
-void Camera::OnMouseButton(const SDL_MouseButtonEvent & event)
+// void Camera::OnMouseButton(const SDL_MouseButtonEvent & event)
```

Comment:
Another dumb mistake on my end: I left this commented bit of code in there on accident 🤦🏾 feel free to remove it!